src/main/java/org/apache/commons/compress/archivers/zip/ZipSplitOutputStream.java

```diff
@@ -57,19 +58,31 @@
      * @param zipFile   the zip file to write to
      * @param splitSize the split size
      */
-    public ZipSplitOutputStream(final File zipFile, final long splitSize) throws IllegalArgumentException, IOException {
+    public ZipSplitOutputStream(final Path zipFile, final long splitSize) throws IllegalArgumentException, IOException {
         if (splitSize < ZIP_SEGMENT_MIN_SIZE || splitSize > ZIP_SEGMENT_MAX_SIZE) {
             throw new IllegalArgumentException("zip split segment size should between 64K and 4,294,967,295");
         }
 
         this.zipFile = zipFile;
         this.splitSize = splitSize;
 
-        this.outputStream = Files.newOutputStream(zipFile.toPath());
+        this.outputStream = Files.newOutputStream(zipFile);
         // write the zip split signature 0x08074B50 to the zip file
         writeZipSplitSignature();
     }
 
+    /**
+     * Create a split zip. If the zip file is smaller than the split size,
+     * then there will only be one split zip, and its suffix is .zip,
+     * otherwise the split segments should be like .z01, .z02, ... .z(N-1), .zip
+     *
+     * @param zipFile   the zip file to write to
+     * @param splitSize the split size
+     */
```

Comment:
Missing since tag on new or protected method.

Reply:
@garydgregory I don't understand your comment, can you elaborate cause you mentioned this kind of change multiple times and I don't want to create more problems.

Reply:
Hi @PostelnicuGeorge 
@garydgregory is talking about the `@since` tag. You can also find this in comments of other methods.
1.21 is already released so you should add a `@since 1.22` here.

Reply:
Thank you so much, on it!